python/TestHarness/resultssummary/summary.py

```diff
@@ -158,6 +158,24 @@ def _format_test_name(test_name):
         """
         return f'`{str(test_name)}`'
 
+    def _sort_key(self, column_index):
```

Comment:
Let's call this `_sort_test_times` because I think that's a bit clearer? Can you also add type hinting?